### PR TITLE
Admin org balance: remove unnecessary reorder

### PR DESCRIPTION
**Pull request by sarthaktexas**
## Summary of the problem

admin balance page is erroring because reorder doesn't work on arrays

## Describe your changes

i removed reorder because sorting is handled inside the filtered_events method and reordering is not needed on it.

### CODE REVIEW

**Review by garyhtou — APPROVED** [review]

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -1030,7 +1030,7 @@ def balances
       flash[:info] = "Do you really want the Start Date to be after the End Date?"
     end
 
-    relation = filtered_events.reorder("events.id asc")
+    relation = filtered_events
     # Omit orgs if they were created after the end date
     relation = relation.where("events.created_at <= ?", @end_date) if @end_date
```
Comment (sampoder) [inline app/controllers/admin_controller.rb]: will `.where` work on the array?

**Comment by sarthaktexas** [comment]
it worked when we tested yesterday!

**Comment by cjdenio** [comment]
it doesn't work— the page breaks if you try to sort by balance + specify an end date: https://hcb.hackclub.com/admin/balances?q=&pending=0&pending=1&unapproved=0&unapproved=1&approved=0&approved=1&rejected=0&rejected=1&transparent=both&omitted=both&demo_mode=full&hidden=both&funded=both&organized_by=anyone&country=&category=&point_of_contact_id=&activity_since=&sort_by=date_desc&start_date=&end_date=2023-10-10&button=

**Comment by sarthaktexas** [comment]
oh my god. i hate this.

**Comment by sampoder** [comment]
lol yeah it's frustrating to get your head around, #4619

**Pull request merged**